config.toml

```diff
@@ -4,10 +4,18 @@ title = "ApacheCon Asia 2021"
 theme = "apachecon"
 # When enabled, creates URL of the form /filename.html instead of /filename/.
 uglyURLs = true
+#hasCJKLanguage = true
+enableRobotsTXT = true
+googleAnalytics = "G-FHFFJCFDG9"
```

Comment:
we just use the UA-192418729-1 here, it is set in themes/apachecon/layouts/partials/header.html.

Reply:
Sorry, forget to remove that, thaks!